core-skills/frontend-web-development/resources/css-architecture.md

```diff
@@ -1,23 +1,67 @@
-- https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout
-- https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Floats 
-- https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox 
-- https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox 
-- https://hacks.mozilla.org/2016/04/you-might-not-need-a-css-framework/
-- https://css-tricks.com/what-are-the-benefits-of-using-a-css-framework/
+# CSS Architecture
 
-## Semantic CSS
+A well-written set of stylesheets balance a number of factors including
+
+* Browser support - does the styling render the same in a wide range of commonly used browsers?
+* Ease of change - can we easily change the styles to meet changing customer needs?
+* Reuse - can we reuse commonly used styles easily?
+* Stability - can we safely change our styles without worrying about unrelated breakages?
+* Expressiveness - can we understand what our styles do, and do they make sense?
+* Performance - are our styles fast to load and render?
+
+If this seems overwhelming, don't worry, we're here to help.
```

Comment:
I'm concerned that it's hard to refer back to how we solve these areas currently